lib/ArtistClass.ts

```diff
@@ -8,25 +8,29 @@ interface ArtistInterface extends Artist {
 
 class Artist {
   artistName: string;
-  firstName: string;
-  lastName: string;
-  location: string;
+  location: {
+    city: string;
+    streetname: string;
+    number: number;
+  };
   slug: string;
   tattoos: string[];
+  position: any; // Not ideal.
```

Comment:
not ideal? sind das die Geodaten zur Position, die nicht nur eine Zahl sind? gibts da was eigenes? ich hab mal Google gefragt, aber ich versteh zu wenig davon, um zu beurteilen ob [dieser Link](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.geolocation.html) oder [der hier](https://stackoverflow.com/questions/39473489/in-typescript-what-is-the-proper-way-to-define-type-which-is-a-subset-of-the-ac) dir in dem Fall was bringen...

Reply:
Die Position ist ein numbers-array. Aber da das array anfangs noch nicht existiert, muss ich es mit null initialisieren. Und dann kann ich der position natürlich nicht den Typ number[] geben... Ich denke das Problem löst sich zusammen mit dem weiter unten.